addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1915,6 +1915,48 @@ options.registry.collapse = options.Class.extend({
     },
 });
 
+options.registry.MasonryLayout = options.Class.extend({
+
+    /**
+     * @constructor
+     */
+     init: function () {
+        this._super.apply(this, arguments);
+        this.$container = this.$('>.container, >.container-fluid, >.o_container_small');
+        this.$currentView = this.$container.children();
```

Comment:
`children()` fails to get text nodes, there should not be any in that case but best gets them with `contents()` anyway (also custo...)